.env.example

```diff
@@ -1,64 +1,68 @@
-# Location and credentials of the postgres database
-DBHOST              = 'postgres'
-DBNAME              = 'genlayer_state'
-DBUSER              = 'postgres'
-DBPASSWORD          = 'postgres'
-DBPORT              = '5432'
+# Postgres Database Configuration
+DBHOST                    = 'postgres'
+DBNAME                    = 'genlayer_state'
+DBUSER                    = 'postgres'
+DBPASSWORD                = 'postgres'
+DBPORT                    = '5432'
 
-# JSON array of initial validators to be created on startup.
-# Example: [{"stake": 100, "provider": "openai", "model": "gpt-4o", "amount": 2}, {"stake": 200, "provider": "anthropic", "model": "claude-3-haiku-20240307", "amount": 1}]
-VALIDATORS_CONFIG_JSON = ''
-
-LOGCONFIG          = 'dev'  # dev/prod
-FLASK_LOG_LEVEL    = 'ERROR'  # DEBUG/INFO/WARNING/ERROR/CRITICAL
+# Logging Configuration
+LOGCONFIG                   = 'dev'     # dev/prod
+FLASK_LOG_LEVEL             = 'ERROR'   # DEBUG/INFO/WARNING/ERROR/CRITICAL
 DISABLE_INFO_LOGS_ENDPOINTS = '["ping", "eth_getTransactionByHash","gen_getContractSchemaForCode","gen_getContractSchema"]'
 
-# JsonRPC server details
+# JsonRPC Server Configuration
 RPCPROTOCOL         = 'http'
 RPCHOST             = 'jsonrpc'
 RPCPORT             = '4000'
-# (the port debugpy is listening on)
-RPCDEBUGPORT        = '4678'
+RPCDEBUGPORT        = '4678'      # debugpy listening port
+JSONRPC_REPLICAS    = '1'         # number of JsonRPC container replicas to run, used to scale up for production
```

Comment:
This is not being used
```suggestion
```

Reply:
But we are planning to scale up JSON-RPC containers in the future to handle multiple users in the studio